src/components/IOUConfirmationList.js

```diff
@@ -101,6 +102,45 @@ const defaultProps = {
 const MINIMUM_BOTTOM_OFFSET = 240;
 
 class IOUConfirmationList extends Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleOption = this.toggleOption.bind(this);
+
+        const formattedSelectedParticipants = this.getFormattedSelectedParticipants(this.props.participants);
+
+        this.state = {
+            selectedParticipants: formattedSelectedParticipants,
+            unselectedParticipants: [],
+        };
+    }
+
+    /**
+     * Returns the selectedParticipants with amount
+     *
+     * @param {Array} selectedParticipants
+     * @returns {Array}
+     */
+    getFormattedSelectedParticipants(selectedParticipants) {
+        return getIOUConfirmationOptionsFromParticipants(
+            selectedParticipants,
+            this.props.numberFormat(this.calculateAmount(selectedParticipants) / 100, {
```

Comment:
We need to pass the second parameter to calculateAmount. Otherwise, none of the participants are listed as the defaultUser, so no one ends up paying the remainder:
<img width="375" alt="Screen Shot 2021-07-08 at 2 06 30 PM" src="https://user-images.githubusercontent.com/49007721/124990827-ba450880-dff5-11eb-981b-89b6e17d5048.png">

Reply:
My bad.
I'll make the split initiator as the defaultUser instead, so that there isn't an extra cent spent.

Reply:
No worries! Sounds good 👍